test/integration/index.test.js

```diff
@@ -1,4 +1,5 @@
 process.env.NODE_ENV = 'test';
+require('dotenv').config();
```

Comment:
Pretty sure I can see from context what's going on here; I appreciate your attention to modularity.